client/src/assets/styles/base.styl

```diff
@@ -0,0 +1,54 @@
+@font-face
+  font-family 'BCSans'
+  font-style normal
+  font-weight 400
+  src url('assets/fonts/BCSans-Regular.woff')
```

Comment:
We provide another font URL for woff2 files for very modern browsers (everything except IE11 I think).